packages/web3-eth/src/web3_eth.ts

```diff
@@ -282,6 +1040,57 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 		return rpcMethodsWrappers.sign(this, message, address, returnFormat);
 	}
 
+	/**
```

Comment:
We should add description that node wallet Account is used for signing in this function via `eth_signTransaction` instead of using local wallet.

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5251/commits/990128fc7f8421a89e6d186e158588189ca37700) commit